content/blog/2022/11/deprecation-policy.md

```diff
@@ -0,0 +1,135 @@
+---
+title:  Keeping track of Kubernetes deprecated resources
+authors:
+- Víctor Cuadrado Juan
+date: 2022-11-04
+---
+
+It's fact of life: As the Kubernetes API evolves, it's periodically reorganized
+or upgraded. This means the old API is [deprecated and later removed](https://kubernetes.io/docs/reference/using-api/deprecation-policy/#deprecating-parts-of-the-api).
+
+We deserve to easily keep track of those deprecations and removals. For that, we
+have just released
+the [deprecated-api-versions policy](https://github.com/kubewarden/deprecated-api-versions-policy/).
+
+### A look at the deprecated-api-versions policy
+
+This policy detects the usage of Kubernetes resources that have been deprecated
+or removed in the API.
+
+It has 2 settings:
+
+- `kubernetes_version`: The starting version from where to detect deprecated or
+  removed Kubernetes resources.
+- `deny_on_deprecation:` If true, it will deny the operation on the resource
+  (normally its creation). If false, it will just log about it. Defaults to
+  true.
```

Comment:
```suggestion
- `deny_on_deprecation:` If true, it will deny the operation on a resource
  that has been deprecated but not yet removed from the Kubernetes version
  specified by `kubernetes_version`. This setting is optional, it is set to `true` by default.
```